content/docs/introduction/getting_started.md

```diff
@@ -37,8 +37,7 @@ Prometheus configuration as a file named `prometheus.yml`:
 ```
 global:
   scrape_interval:     15s # By default, scrape targets every 15 seconds.
-  evaluation_interval: 15s # By default, scrape targets every 15 seconds.
-  # scrape_timeout is set to the global default (10s).
```

Comment:
As this is a getting started document, I found this comment not helpful. The a meaningful timeout is already chosen, no need to confuse new starters about it.